third_party/libpg_query/include/nodes/primnodes.hpp

```diff
@@ -79,6 +79,7 @@ typedef struct PGRangeVar {
 	PGAlias *alias;      /* table alias & optional column aliases */
 	int location;        /* token location, or -1 if unknown */
 	PGNode *sample;      /* sample, if any */
+	bool is_recurring;			 /* is recurring table called */
```

Comment:
Same here - can we move this out of the `PGRangeVar`? I wonder if this is even necessarily - could we not check if `recursive_keys` is set instead?